CHANGELOG.md

```diff
@@ -1,15 +1,127 @@
 # Changelog
 
+## [0.57]
+
+Welcome to the 0.57 release of React Native! This release addresses a number of issues and has some exciting improvements. We've intentionally left this release in an extended release candidate state in order to improve quality. 
+
+You may notice that this release's changelog is thinner than previous versions; that's not from lack of hard work -- this release includes 566 commits by 78 different contributors! Instead, we've taken feedback and prepared a changelog that contains only user impacting changes. Please share your feedback and let us know how we can make this even more useful.
```

Comment:
That's nice! I think one of the ideas that was suggested recently was to make it even thinner focusing on showcasing features one-by-one, in a more React way (as per their blog post) and leaving all the commits at the end by the link. 

To give an example, instead of showing `major improvements` to Accessibility API (line 19) with lots of commits, I would love to see list of improvements instead. Looks like this has been done already in line 11, so that's exactly what I am talking about.

Reply:
@grabbou I like this idea; ultimately, I'd love for the changelog to be absent of commit hashes and authors. In my opinion, that's noise to users.